src/Uno.UI/UI/Xaml/Shapes/Line.iOSmacOS.cs

```diff
@@ -11,6 +11,13 @@ namespace Windows.UI.Xaml.Shapes
 {
 	public partial class Line
 	{
+		public Line()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this redundant constructor.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826527)